src/Symbols/SymbolHelper.cpp

```diff
@@ -249,10 +250,35 @@ ErrorMessageOr<fs::path> SymbolHelper::FindSymbolsFileLocally(
     return ErrorMessage(
         absl::StrFormat("Unable to find symbols in cache for module \"%s\"", module_path.string()));
   }
-  OUTCOME_TRY(VerifySymbolsFile(cache_file_path, build_id));
+  OUTCOME_TRY(verify(cache_file_path));
   return cache_file_path;
 }
 
+ErrorMessageOr<fs::path> SymbolHelper::FindSymbolsInCache(const fs::path& module_path,
+                                                          const std::string& build_id) const {
+  return FindSymbolsInCache(module_path, [&build_id](const std::filesystem::path& cache_file_path) {
+    return VerifySymbolsFile(cache_file_path, build_id);
+  });
+}
+
+ErrorMessageOr<void> SymbolHelper::VerifySymbolsFile(const std::filesystem::path& symbols_path,
+                                                     uint64_t expected_file_size) {
+  if (const uint64_t actual_file_size = std::filesystem::file_size(symbols_path);
```

Comment:
This is the exception throwing variant of std::filesystem::file_size. We do have exception free (and ErrorMessageOr) versions in OrbitBase/File.h